src/app/beer_garden/db/schemas/garden_schema.py

```diff
@@ -0,0 +1,133 @@
+import logging
+
+from brewtils.models import Garden as BrewtilsGarden
+from brewtils.schemas import StatusInfoSchema  # noqa # until we can fully decouple
+from brewtils.schemas import SystemSchema  # noqa # until we can fully decouple
+from marshmallow import Schema, ValidationError, fields
+from marshmallow.decorators import post_load, pre_dump, pre_load, validates_schema
+from mongoengine.queryset.queryset import QuerySet
+
+logger = logging.getLogger(__name__)
+
+
+class GardenBaseSchema(Schema):
+    """Class to give Marshmallow Schemas the desired behavior of throwing
+    exceptions on errors when marshalling/unmarshalling. Otherwise, each line of code
+    utilizing these would need to pull apart MarshalResult objects in order to return
+    a meaningful error."""
+
+    @pre_load
```

Comment:
Is there a reason to call explicitly call this during `pre_load` and `pre_dump`?  It should get naturally called during the load process and I'm not sure you actually want it to run when doing a `dump`...